dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProcessTaskRelationService.java

```diff
@@ -48,5 +48,77 @@
                                                   String conditionType,
                                                   String conditionParams);
 
+
+    /**
+     * query process task relation
+     *
+     * @param loginUser login user
+     * @param projectName project name
+     * @param processDefinitionCode process definition code
+     */
+    Map<String, Object> queryProcessTaskRelation(User loginUser,
+                                                 String projectName,
+                                                 Long processDefinitionCode);
+
+    /**
+     * delete process task relation
+     *
+     * @param loginUser login user
+     * @param projectName project name
+     * @param processDefinitionCode process definition code
+     */
+    Map<String, Object> deleteTaskDefinitionByProcess(User loginUser,
+                                                      String projectName,
+                                                      Long processDefinitionCode);
+
```

Comment:
The function name deleteTaskDefinitionByProcessCode maybe more easy to understand

Reply:
OK